pyasn1_modules/rfc4683.py

```diff
@@ -0,0 +1,70 @@
+#
+# This file is part of pyasn1-modules software.
+#
+# Created by Russ Housley with assistance from asn1ate v.0.6.0.
+#
+# Copyright (c) 2019, Vigil Security, LLC
+# License: http://snmplabs.com/pyasn1/license.html
+#
+# Subject Identification Method (SIM)
+#
+# ASN.1 source from:
+# https://www.rfc-editor.org/rfc/rfc4683.txt
+# https://www.rfc-editor.org/errata/eid1047
+#
+
+from pyasn1.type import univ, char, namedtype, namedval, tag, constraint, useful
```

Comment:
nit: pep8 suggests importing modules one per line

Reply:
Ooops.  I'll fix this one right now.

Reply:
> Ooops. I'll fix this one right now.

Perhaps in another PR then - this one has been merged.

Reply:
Okay.  I'll do another PR.